src/main/frontend/pipeline-graph-view/pipeline-graph/styles/PipelineGraphWidget.scss

```diff
@@ -9,6 +9,16 @@ circle.halo {
   fill: none;
 }
 
+.icon-skipped {
```

Comment:
This class should resolve https://github.com/jenkinsci/pipeline-graph-view-plugin/pull/167#issuecomment-1291172610

<img width="1116" alt="image" src="https://user-images.githubusercontent.com/5103752/198033846-7844ccc0-c00b-4a1b-9ccc-b540c28b98fe.png">

The Jenkins theme has a mixin provided in `icons.less` but I cannot get them to play nice. This will keep the same shape and apply the original color for skipped.

Reply:
Perfect thanks!